Refresh.GameServer/Types/Activity/ActivityPage.cs

```diff
@@ -233,7 +233,6 @@ private void GenerateLevelGroups(ActivityGroups groups)
             levelEvent = @event.EventType switch
             {
                 EventType.LevelUpload => SerializedLevelUploadEvent.FromSerializedLevelEvent(levelEvent),
-                EventType.LevelPlay => SerializedLevelPlayEvent.FromSerializedLevelEvent(levelEvent),
```

Comment:
This and other code needs to be re-added for [this](https://github.com/LittleBigRefresh/Refresh/pull/535/files#diff-e94d1846e9d44e7d024aa109cc13112930742e5fc3b961fa1cd1c7f301fe7922R325) to work again